web/data/data.go

```diff
@@ -111,10 +113,19 @@ func createDoc(c echo.Context) error {
 		return err
 	}
 
+	if err := hooks.Run(instance, realtime.EventCreate, &doc, nil); err != nil {
```

Comment:
Why does this call happen before checking the permissions? And isn't redundant with nearly the same call at line 124?